connector/connector.go

```diff
@@ -43,6 +43,12 @@ type PasswordConnector interface {
 	Login(ctx context.Context, s Scopes, username, password string) (identity Identity, validPassword bool, err error)
 }
 
+// UsernameAttributeProvider allows a connector to override what is displayed
+// as "Username" in the username/password template.
+type UsernameAttributeProvider interface {
+	UsernameAttribute() string
```

Comment:
Just make this part of the PasswordConnector?

```go
type PasswordConnector interface {
    Prompt() (usernameAttribute string)
    Login(ctx context.Context, s Scopes, username, password string) (identity Identity, validPassword bool, err error)
}
```

Reply:
Yeah, that's simpler. Also, thanks for suggesting `Prompt()`, I wasn't happy with `UsernameAttribute` either 😄